desktop/cmp/mask/Mask.js

```diff
@@ -29,6 +29,9 @@ export class Mask extends Component {
 
     static propTypes = {
 
+        /** A model to which a mask is bound to.  Masks for as long as model remains loading. */
+        model: PT.oneOfType([PT.instanceOf(PendingTaskModel), PT.object]),
```

Comment:
In this case, the Mask will always take an actual `PendingTaskModel` instance - not an object.  The usefulness here is that the model can be bound by other models/code to promises, so  it wouldn't do any good if the component created the model on its own and kept it entirely internal.

Also would fixup the comment a bit - how about `Model to which a mask is bound - mask will be visible whenever model.isPending.`